src/main/java/com/github/rholder/retry/WaitStrategies.java

```diff
@@ -310,6 +394,40 @@ public long computeSleepTime(Attempt failedAttempt) {
     }
 
     @Immutable
+    private static final class RandomExponentialWaitStrategy implements WaitStrategy {
+        private static final Random RANDOM = new Random();
```

Comment:
I assume you used a `private static final Random` for consistency with the existing code; I wonder if it's worth switching over to a shared `ThreadLocalRandom`?